tmva/pymva/src/MethodPyKeras.cxx

```diff
@@ -158,10 +160,43 @@ void MethodPyKeras::ProcessOptions() {
    if (fFilenameTrainedModel.IsNull()) {
       fFilenameTrainedModel = GetWeightFileDir() + "/TrainedModel_" + GetName() + ".h5";
    }
+
+   // set here some specific options for Tensorflow backend
+   //  -  when using tensorflow gpu set option to allow memory growth to avoid allocating all memory
+   //  -  set up number of threads for CPU if NumThreads option was specified
+
+   // check first if using tensorflow backend
+   PyRunString("keras_backend_is_tf =  keras.backend.backend() == \"tensorflow\"");
```

Comment:
This could be refactored into a separate function `TMVA::PyKeras::kBackends GetKerasBackend();` to raise the level of abstraction a little bit.

(Possibly just as a named lambda function.)